Dockerfile

```diff
@@ -157,6 +157,9 @@ RUN rustup default ${RUST_VERSION}
 # Install WebAssembly target for Rust.
 RUN rustup target add wasm32-unknown-unknown
 
+# Install musl target for Rust (for statically linked binaries).
+RUN rustup target add x86_64-unknown-linux-musl
```

Comment:
I guess we may need to add something equivalent to the `rust-repository-set` in `WORKSPACE` once we've sorted out #724?

Reply:
I think so